tests/test_wishlist.py

```diff
@@ -54,6 +54,17 @@ def tearDown(self):
         db.session.remove()
         db.drop_all()
 
+    def test_delete_wishlist(self):
+        """ Delete a Wishlist """
+        wishlist = Wishlist(name="wishlist_name", customer_id=1234)
+        wishlist_product = WishlistProduct(wishlist_id=123431, product_id=1213321,
```

Comment:
I don't think this is needed

Reply:
Do you mean the test of the delete?
 Or the product line?  Yes, its needed, I added it to test the cascade delete.

Reply:
Okay. That makes sense. But it is not linked to the wishlist created in the previous line. So it won't get deleted. Also, there is no assert to check if the product got deleted or not.

Reply:
It had an assert also, doesn't it shows? I'll have to check

Reply:
To double check, I also commented out the dropping of the tables and queried them directly.

Reply:
Yeah, I think that assert got lost somewhere. Can you break this up into two tests? One where the cascade delete removes a product and one where it doesn't (different wishlist).

Reply:
The one to delete the product is part of the other story, this is for wishlists. A wishlist with or without will be deleted here.